src/app/portfolio/AssetsBreakdown.tsx

```diff
@@ -71,6 +72,7 @@ const AssetsBreakdownRow: React.FC<{
 export const AssetsBreakdown: React.FC<{
   isLoading: boolean;
   assets: Asset[];
+  stakingPositions?: StakingPosition[]; // Optional to handle undefined case
```

Comment:
No need for very simple comments like this one, you can remove it.
Need to find the sweet spot, to avoid flooding the code with too many comments ^^